packages/flutter/lib/src/widgets/_html_element_view_web.dart

```diff
@@ -123,7 +133,11 @@ class _HtmlElementViewController extends PlatformViewController {
   @override
   Future<void> dispose() async {
     if (_initialized) {
-      await SystemChannels.platform_views.invokeMethod<void>('dispose', viewId);
+      final Map<String, dynamic> args = <String, dynamic>{
+        'platformViewId': viewId,
+        'viewId': flutterViewId,
+      };
+      await SystemChannels.platform_views.invokeMethod<void>('dispose', args);
```

Comment:
I like this better, we should have some rule/lint against passing a scalar value as args for a system channel (at least we can give it a name with a `Map<String, Object?>`

Reply:
Agreed!